docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst

```diff
@@ -14,7 +14,7 @@ To analyze a project, you need to add a :ref:`CodeQL database <codeql-database>`
 
 #. Open the CodeQL Databases view in the sidebar.
 
-#. Hover over the **Databases** title bar and click the appropriate icon to add your database. You can add a database from a local ZIP archive or folder, from a public URL, or from a project slug or URL on LGTM.com.
+#. Hover over the **Databases** title bar and click the appropriate icon to add your database. You can add a database from a local ZIP archive or folder, from a public URL, or from a project slug on GitHub.com.
```

Comment:
> from a project slug on GitHub.com

Do we describe how to do this somewhere? It doesn't seem to be mentioned in the section on getting a database from GitHub.com.

Reply:
I've changed this to "project's URL on GitHub.com" -- the UI is quite clear, so i think that should be sufficient. 

![Screenshot 2022-09-13 at 14 31 47](https://user-images.githubusercontent.com/42464962/189914626-6477b24a-94f9-4fe2-8fd2-116a3d217f62.png)